Eastern-Research-Group/mywaterway
# Feature/improve color accessibility across app

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3342761

## Main Changes:
* Switched to darker Red, Green, and Purple colors for the maps, icons, and badges for better contrast and accessibility. 
* Moved colors into a single shared file where we can easily edit them in the future


## Steps To Test:
1. Navigate to http://localhost:3000/community/020700100102/overview
Verify the Permitted Dischargers on the Overview panel are a gray color and distinguishable.
Verify Identified Issues tab waterbodies are a darker red and the dischargers are gray.

2. Navigate to http://localhost:3000/community/salt%20lake%20city/overview
Verify the lines are clearly visible through the areas.

3. Run Accessibility tools on the waterbody report page, check for Badge contrast ratios to be passing:
http://localhost:3000/waterbody-report/MDE_EASP/MD-02140204
http://localhost:3000/waterbody-report/21AWIC/AL03150110-0202-200

4. Navigate to http://localhost:3000/community/030501110109/overview
Verify points are not black dots and are displayed correctly.

5. Check State page. For Water Quality Overview check that Highcharts is using the new colors and they are legible.
Then check that Advanced Search map is using the darker colors.
http://localhost:3000/state/KS/water-quality-overview
